detection-rules/link_coinbase_phishing_suspicious_link.yml

```diff
@@ -0,0 +1,29 @@
+name: "Brand impersonation: Coinbase with suspicious link"
+description: "This rule detects Coinbase phishing emails with suspicious links"
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  and sender.email.domain.root_domain != "coinbase.com"
+  and length(body.links) == 1 
+  and (
+      regex.icontains(coalesce(sender.display_name, subject.subject),
+        '\b[CⒸⓒ𝑪𝒞𝓬𝔠𝕔𝕥𝖈𝖔𝖙𝗌𝗰𝘊𝘤𝙤𝚌𝝰][oₒⓞ𝑜𝒐𝓸𝕠𝖔𝗼𝘐𝙤𝚘𝞸][i𝐼𝑖𝒊𝓲𝕚𝖎𝗜𝘪𝙄𝚒][nⓃ𝑛𝒏𝓷𝔫𝕟𝖓𝙣𝚗]\s?[BⒷⓑ𝑩𝒃𝓫𝔟𝕓𝕥𝗕][aaₐ𝖆𝗮𝘢𝙖𝚊𝛂𝜶𝞪][s𝑆𝒔𝓢𝔰𝕊𝕊𝖘𝖘𝗦𝘴𝙎𝚂][e𝑒𝒆𝓔𝔢𝕖𝕖𝖊𝖊𝗲𝙀𝚎𝝴𝞮]\b')
```

Comment:
might not be necessary, but we could drop the ascii characters from here and focus on homoglyph characters strictly. if it's ascii it should get picked up by our other coinbase detections. defer to you

Reply:
I think we can probably refactor to coinbase logo too if that's GTG

Reply:
oh yeah we should def work logo detect into at least this one or https://github.com/sublime-security/sublime-rules/blob/6bbce2f381c81d1be6503bf0d720da59cff40bbd/detection-rules/impersonation_coinbase.yml#L4, i thought we had a rule using it but it appears not

Reply:
Coinbase logo detect does work. I tested it last night just need to update the rule. A bunch of the samples though have no attachment so we needed message.screenshot :(